source/shared/core_conn.cpp

```diff
@@ -130,87 +132,64 @@ sqlsrv_conn* core_sqlsrv_connect( _In_ sqlsrv_context& henv_cp, _In_ sqlsrv_cont
          if( options_ht && zend_hash_num_elements( options_ht ) > 0 ) {
          
              zval* option_z = NULL; 
-             option_z = zend_hash_index_find(options_ht, SQLSRV_CONN_OPTION_CONN_POOLING);
+             option_z = zend_hash_index_find( options_ht, SQLSRV_CONN_OPTION_CONN_POOLING );
              if ( option_z ) {
                  // if the option was found and it's not true, then use the non pooled environment handle
                  if(( Z_TYPE_P( option_z ) == IS_STRING && !core_str_zval_is_true( option_z )) || !zend_is_true( option_z ) ) {  
-                henv = &henv_ncp;   
+                henv = &henv_ncp;
+                is_pooled = false;
             }
         }
     }
 #endif // !_WIN32 
 
     SQLHANDLE temp_conn_h;
     core::SQLAllocHandle( SQL_HANDLE_DBC, *henv, &temp_conn_h TSRMLS_CC );
-
     conn = conn_factory( temp_conn_h, err, driver TSRMLS_CC );
     conn->set_func( driver_func );
+    
+    build_connection_string_and_set_conn_attr( conn, server, uid, pwd, options_ht, valid_conn_opts, driver, conn_str TSRMLS_CC );
+    
+    bool missing_driver_error = false;
 
```

Comment:
I know `missing_driver_error` is always there, but consider renaming it as `is_missing_driver` (actually any name that sounds like a boolean flag )